src/api/helper.js

```diff
@@ -0,0 +1,6 @@
+const BASE_URL = 'https://www.omdbapi.com/?apikey=6ce2caab&t=';
+
+export const getTitle = url => (
+  fetch(`${BASE_URL}${url}`)
+    .then(response => response.json())
+);
```

Comment:
Try `async await`
I guess you fetch movie, not title
```suggestion
export const getMovie = url => (
  fetch(`${BASE_URL}${url}`)
    .then(response => response.json())
);
```